.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
shouldn't it be based on release vs tags and the release/tag being used for pypi version too?